package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "imagekit",
-  "version": "4.1.4",
+  "version": "4.1.5",
```

Comment:
We removed some parameters from the code, it needs a major version bump.

Reply:
done